app/src/main/java/org/sopt/at/ui/theme/ShowHidePasswordTextField.kt

```diff
@@ -0,0 +1,82 @@
+package org.sopt.at.ui.theme
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Visibility
+import androidx.compose.material.icons.filled.VisibilityOff
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun ShowHidePasswordTextField(
+    password: String,
+    onPasswordChange: (String) -> Unit
+){
+    var showPassword by remember { mutableStateOf(false) }
+
+    TextField(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 20.dp)
+            .height(55.dp),
+        value = password,
+        onValueChange = {onPasswordChange(it)},
```

Comment:
인자가 없는 경우, 혹은 함수의 인자 순서와 람다의 변수 순서가 같다면 람다 없이 함수이름만 적으셔도 됩니다!
```suggestion
        onValueChange = onPasswordChange
```
가독성이 더 좋아지겠죠?

Reply:
가독성이 훨씬 좋네요!
감사합니다!